pom.xml

```diff
@@ -400,37 +394,16 @@
         <groupId>org.apache.curator</groupId>
         <artifactId>curator-client</artifactId>
         <version>${apache.curator.version}</version>
-        <exclusions>
-          <!-- Excluded to avoid implementation conflict with io.netty:netty-all -->
-          <exclusion>
-            <groupId>org.jboss.netty</groupId>
-            <artifactId>netty</artifactId>
-          </exclusion>
-        </exclusions>
```

Comment:
Although we have removed our explicit dependency on netty, don't several of our dependencies (grpc, zk) transitively depend on netty? Is this safe to remove?

Reply:
They'll bring their own versions. We won't be interfering with their dependencies anymore.

Reply:
don't our dependencies rely on different versions?

Reply:
Current curator version depends on `netty-all`, indirectly via zookeeper. I don't think we need this directive anymore.